runc.go

```diff
@@ -128,7 +128,7 @@ func (r *Runc) Create(context context.Context, id, bundle string, opts *CreateOp
 	cmd.ExtraFiles = opts.ExtraFiles
 
 	if cmd.Stdout == nil && cmd.Stderr == nil {
-		data, err := Monitor.CombinedOutput(cmd)
+		data, err := cmd.CombinedOutput()
```

Comment:
No, you cannot call these directly, you will still have to use `Start` and `Wait` on the monitor.  It will race with the SIGCHLD reaper in containerd.

Reply:
Updated.